src/components/CreateNote/CreateNote.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useReducer } from "react";
+import { v4 as uuid } from "uuid";
+import { useNotes } from "../../context/note-context";
```

Comment:
try to use barrel export

Reply:
I will do this when I have more context. thanks for suggestion